scene/main/viewport.cpp

```diff
@@ -2451,6 +2451,12 @@ void Viewport::_gui_update_mouse_over() {
 		return;
 	}
 
+	if (gui.sending_mouse_enter_exit_notifications) {
+		// If notifications are already being sent, defer call.
+		callable_mp(this, &Viewport::_gui_update_mouse_over).call_deferred();
```

Comment:
In the linked issue, you mention two approaches:
- prevent duplicate update
- defer duplicate update

Are there specific use-cases, that benefit from the deferred call? If there are none, then I would suggest to prevent it.
I understand, that deferring it, might be more robust.

Reply:
So according to your comment, preventing the duplicated update is not a valid approach. Thanks!